lib/bumper.js

```diff
@@ -121,15 +143,21 @@ class Bumper {
   /**
    * Commit the changed files 'package.json' and 'CHANGELOG.md'
    *
+   * @param {Object} options - the command line options
    * @returns {Promise} - a promise resolved with the results of the git commands
    */
-  _commitChanges () {
+  _commitChanges (options) {
     const buildNumber = this.config.ci.buildNumber
 
     return this.ci.setupGitEnv()
       .then(() => {
+        var adds = ['package.json', 'CHANGELOG.md']
+        if (options.blackduck) {
+          adds.push('blackduck/')
```

Comment:
Seems like this should be respecting the option and not hard-coded to `blackduck/`

```js
adds.push(options.blackduck)
```

Although, again, probably better to make it `dependencyDir` or something, so it would end up being:

```js
const adds = ['package.json', 'CHANGELING.md']
if (options.dependencyDir) {
  adds.push(options.dependencyDir)
}
...
```